doc/classes/TabContainer.xml

```diff
@@ -4,8 +4,7 @@
 		Tabbed Container.
 	</brief_description>
 	<description>
-		Tabbed Container. Contains several children controls, but shows only one at the same time. Clicking on the top tabs allows to change the currently visible one.
-		Children controls of this one automatically.
+		Tabbed Container. Toggles the [code]visible[/code] property on all children and sets only the currently selected tab's value to [code]true[/code]. Non-[class Control] children are ignored. A tab will remain visible unless [method set_tab_disabled] removes the image and [method set_tab_title] removes the text. To hide tab content but keep the tab visible, use a [class Control] node as a buffer and hide its children instead.
```

Comment:
Don't reuse the brief description in the description. Avoid passive voice. Avoid future. Link to member variables with [member member_name]. Setters and getters for member variables will disappear from the docs.
Use 's to show possession. The last 2 lines aren't clear. Suggestion:
```
Sets only the active tab's value visible property [code]true[/code]. Hides all other tabs.
Ignores non-[class Control] children. 
One tab is always visible unless you use [method set_tab_disabled] and [method set_tab_title] to toggle its visibility off.
To hide only a tab's content, nest the content inside a child [Control] node and hide it instead of the [code]TabContainer[/code].
```

Reply:
Ah, it seems the docs guide has been updated since I last reviewed it. Thanks.
I was letting other voices slip in too (I'm bad at catching that - already had caught several others).
I was intending to use the `[member member_name]` structure for the `visible` property, but that is a property that is owned by `Control` and I didn't know if `[member Class.member_name]` support had also been implemented. When I asked on Discord, no one responded.
The vast majority of the setters and getters I modified in this class do not correspond to members (except for the tabs_visible/current_tab stuff - woops). Things like `set_tab_disabled/title/icon()` or `get_tab_count()` simply don't have corresponding members, so we should still document those, yes?

Reply:
revised.